container/import.bzl

```diff
@@ -83,6 +83,13 @@ def _container_import_impl(ctx):
         blobsums += [_sha256(ctx, zipped)]
         diff_ids += [_sha256(ctx, unzipped)]
 
+    manifest = ""
+    manifest_digest = ""
+
+    if not (len(ctx.files.manifest) == 0):
```

Comment:
if (len(ctx.files.manifest) > 0): (to make consistent with check in 136)

Reply:
Done